.pre-commit-config.yaml

```
@@ -91,3 +91,16 @@ repos:
         - stylelint-declaration-strict-value@1.9.2
         - postcss-less@3.1.4
       args: [--fix]
+
+  - repo: local
+    hooks:
+    - id: generate-pot
+      name: Generate POT
+      description: This hook generates the .pot file for internationalization.
+      entry: python ./scripts/i18n-messages extract
+      language: python
+      files: '\.(html|py)$'
```

```suggestion
pass_filenames: false
```
This should stop `pre-commit` from running the command multiple times.